src/Components/LayoutType.elm

```diff
@@ -95,6 +95,8 @@ type alias ShowRightSidebarMenu msg =
     , localMsg : msg
     , audioMsg : msg
     , videoMsg : msg
+    , currentlyPlaying : Maybe ItemDetails
+    , playing : Bool
```

Comment:
I think this should be derived from `currentlyPlaying`, if that's `Nothing` it's not playing. So that should be enough in the state?